src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -99,6 +119,26 @@
  *   &lt;property name=&quot;allowLineBreaks&quot; value=&quot;false&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *
+ *   public void dotOperator(String s) {
+ *     Integer.parseInt(s); // Ok
+ *     Integer. parseInt(s); // violation, '.' is followed by whitespace
```

Comment:
Add some code to show allowLineBreaks in action.

Reply:
done